extensions/amazon-lambda/deployment/src/main/java/io/quarkus/amazon/lambda/deployment/AmazonLambdaProcessor.java

```diff
@@ -195,6 +195,12 @@ void ipv4Only(BuildProducer<SystemPropertyBuildItem> systemProperty) {
         systemProperty.produce(new SystemPropertyBuildItem("java.net.preferIPv4Stack", "true"));
     }
 
+    @BuildStep
+    void tmpdirs(BuildProducer<SystemPropertyBuildItem> systemProperty) {
+        systemProperty.produce(new SystemPropertyBuildItem("java.io.tmpdir", "/tmp"));
```

Comment:
I'm not sure this will work in Windows environments

Reply:
Lambda doesn't run on Windows.  But, good point, I should not do this if its test mode.

Reply:
Ok, I just set the properties if we're not in dev or test mode.